src/vex.js

```diff
@@ -221,13 +228,58 @@ var vex = {
     // Content
     var contentEl = vexInstance.contentEl = document.createElement('div')
     contentEl.classList.add(baseClassNames.content)
+    contentEl.setAttribute('role', 'dialog')
+    contentEl.setAttribute('aria-hidden', false)
+    contentEl.setAttribute('aria-labelledBy', options.message)
     addClasses(contentEl, options.contentClassName)
     contentEl.appendChild(options.content instanceof window.Node ? options.content : domify(options.content))
     rootEl.appendChild(contentEl)
+ 
+    //Accessibility features
+    //Get initial element
+    var focusedElementBefore = document.activeElement;
+    
+    document.querySelector(options.ariaContentMain).setAttribute('aria-hidden', true)
+    
+    if (options.trapTabKey) {
+      contentEl.onkeydown = function(event) {        
+        var focusableElementsString = "a[href], area[href], input:not([disabled]), select:not([disabled]), textarea, button, iframe, object, embed, *[tabindex], *[contenteditable]"
+        var focusableElements = contentEl.querySelectorAll(focusableElementsString)
+        
+        //Tab key pressed
+        if (event.which == 9) {
+          console.log(focusableElements.length)
+          var visibleFocusableElements = focusableElements
+                
+          var focusedElement = document.activeElement
+          var numberOfFocusableELements = visibleFocusableElements.length
+                
+          var focusedElementIndex = Array.prototype.indexOf.call(visibleFocusableElements, focusedElement)
+                
+          if (event.shiftKey) {
+            //back tab
+            // if focused on first item and user preses back-tab, go to the last focusable item
+            if (focusedElementIndex == 0) {
+              visibleFocusableElements.item(numberOfFocusableELements - 1).focus()
+              event.preventDefault()
+            }
+          } else {
+            console.log("tab overrided")
+            //forward tab
+            // if focused on the last item and user preses tab, go to the first focusable item
+            if (focusedElementIndex == numberOfFocusableELements - 1) {
+              visibleFocusableElements.item(0).focus();
+              event.preventDefault();
+            }
+          }
+        }
+      }
+    }
 
     // Close button
     if (options.showCloseButton) {
-      var closeEl = vexInstance.closeEl = document.createElement('div')
+      var closeEl = vexInstance.closeEl = document.createElement('button')
```

Comment:
Just a sanity check on our CSS, does this break any styles?

Reply:
This changed the appearance of several styles so it was changed back to div